src/AccessControl/cAccessControl.c

```diff
@@ -484,7 +484,7 @@ static PyExtensionClass SecurityManagerType = {
 	0,					/* tp_itemsize	*/
 	/* Standard methods 	*/
 	(destructor) SecurityManager_dealloc,/* tp_dealloc	*/
-	NULL,					/* tp_print	*/
+	0,  					/* tp_print	*/
```

Comment:
Apparently, I forgot this one.
```suggestion
	0,  					/* tp_vectorcall_offset*/
```